src/vaishnaviVJun20_arrayExamples/PracticeExamTwo.java

```diff
@@ -0,0 +1,23 @@
+package vaishnaviVJun20_arrayExamples;
+
+public class PracticeExamTwo {
+	
+	void findSumEqualToGivenNumber(int[] array1,int num) {
+		System.out.println("Integer numbers, whose sum is equal to value: "+num);
+		 for(int index=0;index<array1.length;index++) {
+			for(int i=1;i<array1.length;i++) {
+			   if(array1[index]+array1[i] == num) {				   
```

Comment:
This is correct but to safer side always do () for mathematics operation in java, in your code line number 9

Reply:
Thank you. I will keep this in mind.